packages/contentstack-config/package.json

```diff
@@ -20,15 +20,15 @@
     "test:unit": "mocha --forbid-only \"test/unit/*.test.ts\" --unit-test"
   },
   "dependencies": {
-    "@contentstack/cli-command": "^1.1.0",
-    "@contentstack/cli-utilities": "^1.1.0",
+    "@contentstack/cli-command": "^1.2.0",
+    "@contentstack/cli-utilities": "^1.2.0",
+    "@oclif/core": "^2.3.0",
```

Comment:
@aman19K we can remove this package if not used directly

Reply:
@antonyagustine removed @oclif/core dependency from package.json